src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -3098,6 +3098,53 @@ public string AwsAccountId
         }
         #endregion
 
+        #region Otel Bridge
+
+        // TODO: configure the initial list of activity sources to include
+        private static readonly string[] DefaultIncludedActivitySources = ["NewRelic.Agent", "TestApp activity source"];
```

Comment:
I think that this list should only include the `NewRelic.Agent` source for now, and that the hybrid agent cross agent tests should use the configuration to add the `TestApp activity source`.